operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1323,8 +1323,8 @@ spec:
   relatedImages:
```

Comment:
There may be a new openJDK image available but we will have to be careful about updating the version in order the MD5 problem we had last time. We may not need to update anything here. Feel free to save this change for another PR!

Reply:
wdym? We are going anyway to remove Kafka 3.0.0 to be replaced by 3.2.0. The SHAs are not the final ones of course.

Reply:
Talking about the strimzi-maven-builder image int the relatedImages section but yes let's save the change for when we update the other MD5s